### PR TITLE
[fix] Sync packaging helpers doc french → english

### PR DESCRIPTION
**Pull request by JocelynDelalande**
The english version was massively out-of-date. It now provides the same
information as the french version.

It's a big text. Despite my own review, additional eyeballs on this translation are more than welcome :-)

### CODE REVIEW

**Comment by alexAubin** [comment]
Thanks for this !

Related issue on redmine : https://dev.yunohost.org/issues/870

File: packaging_apps_helpers.md [hunk 1]
```diff
@@ -2,85 +2,264 @@
...
+ynh_package_version PACKAGE
+```
+> Returns the installed version number of `PACKAGE`.
+
+```bash
+ynh_package_update
+```
+> Updates the packages list (`apt update`) in a silent and non-interactive way.
+
+**Be careful, the following commands are to be avoided when possible. Installing (and even more removing) a package without handling conflicts and dependencies. That will be improved in future Yunohost versions.**
```
Comment (alexAubin) [inline packaging_apps_helpers.md]: Isn't a verb lacking at the end of the middle sentence ? Like "installing a package without handling conflicts and dependencies **is not recommended**" ?

**Comment by JocelynDelalande** [comment]
> Related issue on redmine : https://dev.yunohost.org/issues/870

That's fun, I didn't see your issue from yesterday :)

File: packaging_apps_helpers.md (hunk 1 above)
Comment (JocelynDelalande) [inline packaging_apps_helpers.md]: @alexAubin agreed. I change that.

**Pull request merged**